variant/translators/translator.py

```diff
@@ -22,8 +22,19 @@ def translate(self, res: ValidationResult) -> Allele:
         """Translate to VRS representation."""
         instance_tokens = [t for t in res.classification.all_tokens if
                            self.is_token_instance(t)]
+        len_instance_tokens = len(instance_tokens)
+
+        if len_instance_tokens > 1:
+            if len_instance_tokens == 2:
```

Comment:
Couldn't you just delete line 27?

Reply:
No. Line 28 is a case where you have a query like `NP_000542.1:p.Tyr185Ter (p.Tyr185*)`, where you get two PolypeptideTruncation tokens. This is okay and should really be the only time, as of now, where you expect two of the same instance tokens, else we want to raise an exception